### PR TITLE
Nuke Ops Overhaul Part 1: Blastco

### PR DESCRIPTION
**Pull request by Chronitonity**
Mainly an addition of the (Very fucking rough.) base for the rest of the
nuke ops teams.

This part adds Blast Co, each of them get a new hardsuit (Sprites
courtesy of Wjohn)
A grenade launcher which slows them down and some ammo , a macrobomb, a
drop beacon for a normal bomb,  2 minibombs, 3 c4's, an incen grenade,
rollerskates to speed you up, and a pinpointer.

Also adds a sniper and chem bundle for nuke ops, and fixes slowdown for
inhands.

### CODE REVIEW

**Comment by Foosili** [comment]
hoyl SHITTTTTT BOY i DIE BOY :+1: :+1: :100: :100:

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -217,6 +230,20 @@ var/list/uplink_items = list()
...
 
+/datum/uplink_item/specoffer/sniper
+	name = "Sniper bundle"
+	desc = "Contains a collapsed sniper rifle in an expensive carrying case, a hollowpoint haemorrhage magazine, a soporific knockout magazine, a free surplus supressor, and a worn out suit and tie."
+	item = /obj/item/weapon/storage/briefcase/sniperbundle
+	cost = 20//26 normal cost. suppressor is excluded from price.
+	gamemodes = list(/datum/game_mode/nuclear)
+
+/datum/uplink_item/specoffer/chemical
+	name = "Tiger Cooperation Chemical Bioterror bundle"
```
Comment (Weisyl) [inline code/datums/uplink_item.dm]: Should be Tiger Corporation instead of Co-o-peration

Comment (Chronitonity) [inline code/datums/uplink_item.dm]: oof

**Comment by Lawligagger** [comment]
Oh right, I remember this from FP code

+1

**Comment by SplergeBucket** [comment]
yeah its good

**Pull request merged**